x/wasm/relay_test.go

```diff
@@ -497,6 +557,43 @@ func (c *sendEmulatedIBCTransferContract) IBCPacketTimeout(codeID wasmvm.Checksu
 	return &wasmvmtypes.IBCBasicResponse{Messages: []wasmvmtypes.SubMsg{{ReplyOn: wasmvmtypes.ReplyNever, Msg: wasmvmtypes.CosmosMsg{Bank: returnTokens}}}}, 0, nil
 }
 
+var _ wasmtesting.IBCContractCallbacks = &captureCloseContractNotOwned{}
+
+type captureCloseContractNotOwned struct {
```

Comment:
Nit: it is not capturing anything. How about:
```suggestion
type closeChannelContract struct {
```

Reply:
I've changed this